samples/zephyr/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -138,6 +139,7 @@ tests:
       - SB_CONFIG_SECURE_BOOT_APPCORE=y
       - SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU=y
       - SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519=y
+      - SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE=y
```

Comment:
`SB_CONFIG_SECURE_BOOT_GENERATE_DEFAULT_KMU_KEYFILE=y` must be also added for NSIB,
However, this is fixed in PR https://github.com/nrfconnect/sdk-nrf/pull/25203 by adding Kconfig.sysbuild